js/viz/series/base_series.js

```diff
@@ -543,12 +565,14 @@ Series.prototype = {
 
         that._appendInGroup();
 
-        that._applyVisibleArea();
-        that._setGroupsSettings(animationEnabled, firstDrawing);
-
-        !firstDrawing && !that._resetApplyingAnimation && that._drawElements(false, firstDrawing, false);
-        that._drawElements(animationEnabled, firstDrawing, true);
+        if(!that._isAllPointsTranslated) {
+            that.prepareCoordinatesForPoints();
+        }
 
+        that._isAllPointsTranslated = false;
```

Comment:
We need to translate the points when rotating series
(We have written a test for this "Visible area, rotated")

If I do not do this, we will get an error because we will not rebroadcast the points